tests/gpaw/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+# add tests for GPAW4QMCPACK converter
+function(ADD_GPAW4QMCPACK_TEST test_name)
+      set(EXE_NAME ${qmcpack_BINARY_DIR}/bin/gpaw4qmcpack)
+  execute_process(COMMAND ${CMAKE_COMMAND} -E copy "${CMAKE_CURRENT_SOURCE_DIR}/gpaw4qmcpack_test.py"
+                          "${CMAKE_CURRENT_BINARY_DIR}")
+  execute_process(COMMAND ${CMAKE_COMMAND} -E copy_directory "${CMAKE_CURRENT_SOURCE_DIR}/${test_name}"
```

Comment:
Better to create the destination folder softlink necessary files instead of copy.

Reply:
You're probably right. I just copied this from the existing converter tests in `tests/converter/.` Let me look into it.

Reply:
So let me make sure: first create an actual destination directory, then soft link the necessary contents? If I soft link the whole directory it will clutter the source code upon testing, right?

Reply:
Unfortuantely, we can not softlink the folder. Multiple build may share the same source folder.

Reply:
I now create a new folder and soft link the files needed for testing